src/themes/elife/styles.css

```diff
@@ -15,6 +15,9 @@
 
 @import '../../extensions/person/styles.css';
 
+/* Custom Properties */
```

Comment:
Comments are good, but in this case the file name is self-documenting so don't need this here.